src/com/google/javascript/jscomp/Compiler.java

```diff
@@ -672,6 +672,11 @@ private void compileInternal() {
     if (!precheck()) {
       return;
     }
+    
+    if (options.skipNonTranspilationPasses) { // TODO: add clause "|| options.closurePass"?
```

Comment:
that's meant to be:
```javascript
// TODO: add clause "|| !options.closurePass"?
```
I'm asking is there a case where skipNonTranspilationPasses=false but we still need this goog.module wrapping?